### ISSUE TITLE
Launching programmes with custom keys not working with the latest update

### ISSUE THREAD

**Issue by jtmart** [seerge/g-helper#1300]
Hi,

On my flow x16+GHelper I mapped M4 to cmd.exe (C:\Windows\System32\cmd.exe) and it used to work fine till the last update. Now, however, this seems broken.

Any idea how to fix the issue?

Best,

Jean-Thomas

**Comment by jtmart** [comment]
I used to just open an empty console without any particular command. The custom key was C:\Windows\System32\cmd.exe. Tried %windir%\system32\cmd.exe with no avail...

**Comment by seerge** [comment]
@jtmart it's because to accommodate any number of arguments for a command itself, i'm now launching everything via "cmd /c" itself

So for your case you actually need to start cmd.exe from cmd.exe :)

Try to use following as a command ```start cmd.exe```

**Comment by jtmart** [comment]
Yes, this fixed it @seerge! Cmd /c (or /k?) is a great idea; thank you for implementing this!
